client/Sheepshead/SharedSheep/Round/IRound.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SharedSheep.Player;
+using SharedSheep.Trick;
+namespace SharedSheep.Round
+{
+    interface IRound
+    {
+        ITrick Trick { get; set; }
+        int RoundNumber { get; set; }
+        IPlayer RoundStarter { get; set; }
```

Comment:
Should anyone except the Round instance be able to change these variables after a round has started? Maybe remove the setters from the interface?